app/scenes/channel_info/channel_info.js

```diff
@@ -169,23 +171,27 @@ class ChannelInfo extends PureComponent {
                     </View>
                     <ChannelInfoRow
                         action={this.props.actions.goToChannelMembers}
-                        defaultMessage='Manage Members'
+                        defaultMessage={isAdmin ? 'Manage Members' : 'View Members'}
```

Comment:
@enahum Have you discussed always using "View Members" with UX? I know you mentioned doing this at some point